comtypes/_comobject.py

```diff
@@ -435,21 +533,24 @@ def __make_interface_pointer(self, itf):
         # iterate over interface inheritance in reverse order to build the
         # virtual function table, and leave out the 'object' base class.
         finder = self._get_method_finder_(itf)
+        interface = None
```

Comment:
same as line 575 in `comtypes/__init__.py`

Reply:
If that is removed there is the possibility of interface not being defined and it would result in a NameError. The only way interface would get defined is from the for loop and if there is nothing to iterate over the variable would not get made.